tests/python/unittest/test_sparse_operator.py

```diff
@@ -2098,6 +2098,77 @@ def check_scatter_ops(name, shape, lhs_stype, rhs_stype, forward_mxnet_call, for
                           lambda l, r: l + r,
                           rhs_is_scalar=True, verbose=False, density=0.5)
 
+
+@with_seed()
```

Comment:
It will be great to add a reference to test_batchnorm_training in test_operator.py, and note this is to test row_sparse fallback correctness. Just to avoid confusion for other developers who work on batchnorm.